cmd/configschema/go.mod

```diff
@@ -348,7 +348,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.45.1 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.45.1 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.45.1 // indirect
-	github.com/open-telemetry/opentelemetry-log-collection v0.24.0 // indirect
+	github.com/open-telemetry/opentelemetry-log-collection v0.25.0 // indirect
```

Comment:
Btw why v0.25.0 is marked as pre-release?

Reply:
No longer is. I should have unchecked that before asking for review here. I just wanted to see all CI pass here before marking it official.